components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -216,6 +218,9 @@ public class OAuth2AuthzEndpoint {
 
     private static final String OIDC_DIALECT = "http://wso2.org/oidc/claim";
 
+    private static final String defaultOAuthAuthzRequestClassName =
```

Comment:
Since the default class is defined in the OAuthServerConfiguration class let's get it from there. No need to define here.

Reply:
Fixed with 79bab73a362a2021f071c540d7aed7a385ff3817